www/grpc/buf/buf.gen.yaml

```diff
@@ -32,6 +32,9 @@ plugins:
     opt:
       - grpc_api_configuration=grpc-gateway.config.yaml,allow_merge=true,merge_file_name=pactus
       - openapi_configuration=openapi.config.yaml
+  - name: jrpc-gateway
+    out: ../gen/go
+    opt: paths=source_relative
```

Comment:
Is this option required?

Reply:
For now, it won't be required. I put it there for future plugin needs.

Reply:
So, let's remove it and add it when we need it

Reply:
Ok